### ISSUE TITLE
import - [js] Expression expected

### ISSUE THREAD

**Issue by victorhugoweb** [eslint/eslint#8692]
In VScode  shows this error:  [js] Expression expected

![sem titulo](https://user-images.githubusercontent.com/29238083/26849122-58e5b450-4ad9-11e7-8fcb-2ffef9dc1ddb.png)
 
 
  

```js


const loadSobre = () => import('./paginas/sobre'  );


```

**Comment by eslintbot** [comment]
Hi @victorhugoweb, thanks for the issue. It looks like there's not enough information for us to know how to help you.

If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

Requesting a rule change? Please see [Proposing a Rule Change](http://eslint.org/docs/developer-guide/contributing/rule-changes) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!

**Comment by mysticatea** [comment]
Thank you for this issue.

However, this is not related to ESLint. It's the validation of VSCode itself. You can disable the validation by your VSCode setting.
